console-subscriber/src/builder.rs

```diff
@@ -149,6 +159,279 @@ impl Builder {
 
         self
     }
+
+    /// Initializes the console [tracing `Subscriber`][sub] and starts the console
+    /// subscriber [`Server`] on its own background thread.
+    ///
+    /// This function represents the easiest way to get started using
+    /// tokio-console.
+    ///
+    /// In addition to the [`TasksLayer`], which collects instrumentation data
+    /// consumed by the console, the default [`Subscriber`][sub] initialized by this
+    /// function also includes a [`tracing_subscriber::fmt`] layer, which logs
+    /// tracing spans and events to stdout. Which spans and events are logged will
+    /// be determined by the `RUST_LOG` environment variable.
+    ///
+    /// **Note**: this function sets the [default `tracing` subscriber][default]
+    /// for your application. If you need to add additional layers to this
+    /// subscriber, see [`build`].
```

Comment:
this should probably refer to the new `build_console_layer` method now (which i think we may want to rename to `spawn` or something...)